src/server.js

```diff
@@ -0,0 +1,84 @@
+require('dotenv').load();
+
+const AccessToken = require('twilio').jwt.AccessToken;
+const VoiceGrant = AccessToken.VoiceGrant;
+const VoiceResponse = require('twilio').twiml.VoiceResponse;
+
+function tokenGenerator(identity) {
+  // Used when generating any kind of tokens
+  const accountSid = process.env.ACCOUNT_SID;
+  const apiKey = process.env.API_KEY;
+  const apiSecret = process.env.API_KEY_SECRET;
+
+  // Used specifically for creating Voice tokens
+  const pushCredSid = process.env.PUSH_CREDENTIAL_SID;
+  const outgoingApplicationSid = process.env.APP_SID;
+
+  // Create an access token which we will sign and return to the client,
+  // containing the grant we just created
+  const voiceGrant = new VoiceGrant({
+      outgoingApplicationSid: outgoingApplicationSid,
+      pushCredentialSid: pushCredSid
+    });
+
+  // Create an access token which we will sign and return to the client,
+  // containing the grant we just created
+  const token = new AccessToken(accountSid, apiKey, apiSecret);
+  token.addGrant(voiceGrant);
+  token.identity = identity;
+
+  // Serialize the token to a JWT string
+  return token.toJwt();
+}
+
+function makeCall(to) {
+  // Use a valid Twilio number by adding to your account via https://www.twilio.com/console/phone-numbers/verified
+  defaultCallerId = "1234567890";
```

Comment:
Let's create constants at the top for `IDENTITY`, `CALLER_ID`, and `CALLER_NUMBER` like in https://github.com/twilio/voice-quickstart-server-python/pull/23

Not sure if it should be CAPS case. Use the language idiomatic way of defining const variables